### ISSUE TITLE
BQ.1 sublineage circualting in China with Orf1a:A2033V and Orf1a:E754K

### ISSUE THREAD

**Issue by FedeGueli** [cov-lineages/pango-designation#1559]
There may be a lineage circualting in china sampled  on planes from China directed to Italy and France.

Tree:
![Schermata 2023-01-17 alle 09 26 01](https://user-images.githubusercontent.com/87669813/212846525-070943c4-3563-4e1d-aee0-8df7b53863bb.png)
https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_251da_65aec0.json?c=gt-ORF1ab_2033,754&label=id:node_6395123

Defining mutations: S:436R rev  G22599C > A17039G > C22599G > C28603T > T27546C > Orf1a:A2033V (C6363T) >Orf1a:E754K ( G2525A)

**Comment by FedeGueli** [comment]
2 more sample from Israel

**Comment by AngieHinrichs** [comment]
> Is this BQ.1 or BQ.1.1 ? in other words, it is 346R a reversion or a wild-type?

I believe it is BQ.1 and the apparent reversion at 22599 (S:346) is a tree-building error.  It is tricky to reconcile the maximum parsimony approach with recurring mutations that confer a growth advantage like G22599C (S:R346T).  Sorry that it makes it harder to identify the real parental lineage in cases like this.  

It seems most likely that G22599C happened multiple times, for example on BQ.1 > A17039G (BQ.1.1) and also directly on BQ.1 (BQ.1.19).  But it's equally parsimonious to suppose that G22599C happened only once, and then A17039G afterwards, and then to explain the existence of some sequences with A17039G but not G22599C by making a reversion on 22599.  🙁  

I recently added the label BQ.1_17039 to the branch with the reversion on 22599 so that sequences on that branch would not be falsely assigned to BQ.1.1 in the minimized-for-pangolin tree.  Whenever you see a label with a real lineage followed by '`_`' and something cryptic, its purpose is to prevent bad assignments in pangolin.  In the minimized tree for pangolin, the '`_`' and anything that follows it is removed from the label -- so there are multiple labels for BQ.1 and some other lineages.  In the daily build, I need to maintain unique labels so that each label is carried forward from one day to the next, so we get the ugly labels like BQ.1_17039 when I disagree with the order of mutations in the tree but can't convince matOptimize to change them by removing a few stray sequences (there are just too many sequences with G22599C!).

This is happening more and more frequently with the trend toward convergent evolution with advantageous Spike mutations recurring and spreading.  We should look into enhancing matUtils (or matOptimize?) to recognize this pattern (mutations A > B > C > revB) and fix it (move A > B > C > revB to A > C, and move A > B > C to A > C > B).

**Comment by FedeGueli** [comment]
Thank you @AngieHinrichs  now i ve corrected  title and main post!

**Comment by aviczhl2** [comment]
Most of the sequences are from Europe local transmissions starting from October. Others are from USA or Mexico local sequences starting from late Nov.

None of them are from travelers from China to Japan, South Korea or Singapore. Also none from Chinese local provincial CDCs. 

Guess this lineage is more likely to be related to Europe-related planes or European airports.

**Comment by FedeGueli** [comment]
> Most of the sequences are from Europe local transmissions starting from October. Others are from USA or Mexico local sequences starting from late Nov.
> 
> None of them are from travelers from China to Japan, South Korea or Singapore. Also none from Chinese local provincial CDCs.
> 
> Guess this lineage is more likely to be related to Europe-related planes or European airports.

Yes that was my thought too. Lets see if some new samplesshow up in the next way

**Comment by FedeGueli** [comment]
It doesnt seem to circulate in China as today closing this .
